docs/GettingStarted.md

```diff
@@ -300,8 +298,36 @@ You can also get the root span of the current active trace using the `active_roo
 current_root_span = Datadog.tracer.active_root_span
 current_root_span.set_tag('my_tag', 'my_value') unless current_root_span.nil?
 ```
+## Compatibility
+
+### Interpreter Compatibility
+
+Ruby APM includes support for the following Ruby interpreters:
+
+
+| Type                               | Version | Support type    |
+| ---------------------------------- | -----   | --------------- |
+| [MRI][10]  | 1.9.1   | Experimental    |
+|                                    | 1.9.3   | Fully Supported |
+|                                    | 2.0     | Fully Supported |
+|                                    | 2.1     | Fully Supported |
+|                                    | 2.2     | Fully Supported |
+|                                    | 2.3     | Fully Supported |
+|                                    | 2.4     | Fully Supported |
+|                                    | 2.5     | Fully Supported |
+| [JRuby][11]         | 9.1.5   | Experimental    |
+
+### Web Server Compatibility
+
+Ruby APM includes support for the following web servers:
+
+| Type                                           | Version      | Support type    |
+| ---------------------------------------------- | ------------ | --------------- |
+| [Puma][12]                        | 2.16+ / 3.6+ | Fully Supported |
+| [Unicorn][13]       | 4.8+ / 5.1+  | Fully Supported |
+| [Passenger][14] | 5.0+         | Fully Supported |
 
-## Integration instrumentation
+### Library and Framework Compatibility
```

Comment:
Changing this name breaks anchor at top of document. I recommend keeping `Integration instrumentation` since it looks nice seeing `Manual` then `Integration`.

But if it isn't clear what "integration" means, maybe renaming to `Library and Framework Instrumentation` (an updating the anchor at the top) would be a good compromise.

Reply:
I think this was me trying to combine the two pages. Looking at it with some distance, you're right that `Integration instrumentation` works best here. Changing back.